requirements.txt

```diff
@@ -3,8 +3,8 @@ attrdict==2.0.0
 eventlet==0.19.0
 python-socketio==1.6.1
 numpy==1.13.1
-Pillow==2.2.1
+Pillow==6.1.0
 scipy==0.19.1
 keras==2.0.8
-tensorflow==1.3.0
+tensorflow==1.14.0
```

Comment:
Why are we upgrading tensorflow? If we start using a version that is higher than 1.3.0, we may not be able to run our code on Carla...

Reply:
So, we should probably change this back to 1.3.0

Reply:
These models don't support old versions of tensorflow. I actually supposed that this docker image will be run on carla as it is, so it will update dependencies, and run our app with new dependencies.  But if I'm wrong can we confirm it from udacity resources?
PS. I checked all the first repos or Capstone in google search. It looks like you're right. But better to confirm this.

Reply:
In Marcomarasca speaks about "Converting for Tensorflow 1.3" in his README. I assume, that in "converted" or "converted_optimized" folders are compatible with the 1.3 tf library version.

Reply:
In the past I've had problems using updated TensorFlow versions with Udacity projects, and ended up with compatibility issues, I would stay with 1.3.

Reply:
ok

Reply:
@bessszilard 
thanks for hint, I already used optimised model, but didn't downgrade the version. Now it's fixed